scripts/exgfs_atmos_post.sh

```diff
@@ -206,9 +206,9 @@ if [ "${stime}" = "anl" ]; then
           wafsgrid="latlon 0:1440:0.25 90:721:-0.25"
           ${WGRIB2} "${PGBOUT}" -set_grib_type same -new_grid_winds earth \
           -new_grid_interpolation bilinear -set_bitmap 1 \
-          -new_grid ${wafsgrid} "${PGBOUT}.tmp"
+          -new_grid "${wafsgrid}" "${PGBOUT}.tmp"
```

Comment:
These quotes will break the command. `${wafsgrid}` intentionally contains multiple arguments. Please ignore the linter warning for this line.